src/global/notifications/notifications.android.ts

```diff
@@ -346,12 +342,14 @@ class NotificationsAndroid extends NativeEventEmitterComponent<NotificationEvent
   }
   registerForPushNotifications(onSuccess, onFailure) {
     if (!AndroidConfig.isEmulator) {
-      registerPushNotification(onSuccess, onFailure);
+      registerPushNotification.bind(this, onSuccess, onFailure);
     } else {
       onFailure && onFailure();
     }
   }
-  static LocalNotification = LocalNotification;
+  LocalNotification = LocalNotification;
 }
 
-export default NotificationsAndroid;
+const Notifications = new NotificationsAndroid();
```

Comment:
Change the name of the variable to NotificationsAndroid and change the name of the class to NotificationAndroidClass so that while exporting, users wouldn't confuse those.